test/definitions/divorce/CaseTypeTab.test.js

```diff
@@ -0,0 +1,112 @@
+const expect = require('chai').expect;
+const assert = require('chai').assert;
+const _ = require('loadsh');
+const caseTypeTab = require('../../../definitions/divorce/json/CaseTypeTab');
```

Comment:
Might be a good idea to clone this to keep the original data immutable.

Reply:
How to?

Reply:
```
const caseTypeTab = Object.assign({}, require('../../../definitions/divorce/json/CaseTypeTab'));
```

Reply:
👍